manifests/alertmanager.pp

```diff
@@ -103,6 +109,7 @@
   String[1] $package_name,
   Array $receivers,
   Hash $route,
+  Array $mute_time_intervals,
```

Comment:
can we enforce the datatype a bit more? is it always an array of hashes?

Reply:
Yes, this is always an Array of Hashes, I adjusted the data type to reflect that.